discord-readme.md

```diff
@@ -45,11 +46,11 @@ GENERAL DISCUSSION:
 
 -> #content-share: share web content
 
--> #bot-chat: discuss bots, esp. regarding development of EddieBot
+-> #bot-chat: discuss bots, esp. regarding development of EddieBot. This is also a place to check your profile and ranks in the community using the bot.
 
 DAILY:
 
--> #healthfitness: health and fitness because you want to get those biceps
+-> #healthfitness: share your tips on health and fitness so we can be healthfitness together.

```

Comment:
```suggestion
-> #healthfitness: share your tips on health and fitness so we can be healthy together.
```
`healthfitness` isn't really a word. Maybe "so we can be healthy together"?

Reply:
Yeah, I agree with you. I'm just trying to match with the name's channel. It seems odd if we use the word healthfitness.